### DISCUSSION TITLE
windowresize-issue when display turns off and on again (macOS Client)

### DISCUSSION THREAD

**Discussion by jan-the-maker** [rustdesk/rustdesk#9932] [General]
### Bug Description

I attached a screenshot which shows what happens when I turn off my display and turn it on again.

### How to Reproduce

On MacOs (14.7 Sonoma) put a Rustdesk Window in Fullscreen - then turn off the display and turn it back on.

### Expected Behavior

The window should cover the fullscreen as it did before the display went off.

### Operating system(s) on local side and remote side

macOS 14.7 > Win11

### RustDesk Version(s) on local side and remote side

1.3.1 -> 1.3.1

### Screenshots

![Screenshot 2024-11-16 at 14 11 46](https://github.com/user-attachments/assets/d5ee1fe0-57ec-4c31-9ebf-a787ffbd8e82)

**Comment by jan-the-maker** [comment]
My MacOS background is black. I just noticed it's hard to see. Sorry for that. Please click the screenshot ... it should become clear then.

**Comment by rustdesk** [comment]
Sorry, I have to lower this case by moving to discussion because of this corner case, if we receive more reports, we will take action.

**Comment by jan-the-maker** [comment]
No problem, just letting you know. Love RustDesk and if that's the only issue I gladly take it :)

**Reply by rustdesk** [reply]
I just turned off and turned on again on mac, the full screen remote window was normal, nothing changed.

**Reply by jan-the-maker** [reply]
macOS ARM? HDMI connected?

**Reply by jan-the-maker** [reply]
ah maybe its my HDMI switch ... I will test it later without to be sure
